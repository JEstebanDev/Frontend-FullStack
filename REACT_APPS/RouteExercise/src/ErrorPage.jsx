import React from "react";

export default function ErrorPage() {
  return (
    <div>
      <h2>Ops!</h2>
      <span>Are you lost?</span>
    </div>
  );
}
