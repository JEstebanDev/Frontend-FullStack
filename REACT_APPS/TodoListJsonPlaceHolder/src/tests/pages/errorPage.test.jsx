import { render } from "@testing-library/react";
import ErrorPage from "../../pages/errorPage";
import { screen } from "@testing-library/react";
describe("testing the component", () => {
  test("should render the ErrorPage and take snapshot", () => {
    const { container } = render(<ErrorPage></ErrorPage>);
    expect(container).toMatchSnapshot();
  });

  test("should render the Title in ErrorPage", () => {
    render(<ErrorPage></ErrorPage>);
    screen.getByTestId("message");
  });
});
