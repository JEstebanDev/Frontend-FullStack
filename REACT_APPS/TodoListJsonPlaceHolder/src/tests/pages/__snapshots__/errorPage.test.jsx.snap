// Vitest Snapshot v1

exports[`testing the component > should render the ErrorPage and take snapshot 1`] = `
<div>
  <div
    class="card info-form "
  >
    <h3
      data-testid="message"
    >
      Oops!
    </h3>
    <p>
      are you lost?
    </p>
  </div>
</div>
`;
